app-misc/sl/files/Makefile-r4

```diff
@@ -0,0 +1,5 @@
+LDLIBS = `pkg-config --libs ncurses`
```

Comment:
You have to use `$(PKG_CONFIG)` here to allow override, and use tc-getPKG_CONFIG in the ebuild. However, it seems a lot of effort to add a Makefile for this, so you can instead do something like:

```
src_compile() {
    emake LDLIBS="$($(tc-getPKG_CONFIG) --libs ncurses)"
}
```

to override the variable in the old Makefile from ebuild.